### PR TITLE
TRM-29300: Update to use spark UniProtEntryConverter in Integration t…

### PR DESCRIPTION
**Pull request by LeonardoGonzales**
…ests

### CODE REVIEW

File: common-rest/pom.xml [hunk 1]
```diff
@@ -223,13 +223,6 @@
...
 			<scope>test</scope>
 		</dependency>
-		<dependency>
-			<groupId>org.uniprot</groupId>
-			<artifactId>uniprot-indexer-integration-test</artifactId>
-			<version>${uniprot-store.version}</version>
-			<type>test-jar</type>
-			<scope>test</scope>
-		</dependency>
 		<dependency>
```
Comment (ahmadshadab) [inline common-rest/pom.xml:226]: good God we removed this. thanks

File: uniprotkb-rest/src/test/java/org/uniprot/api/uniprotkb/controller/TestUtils.java [hunk 2]
```diff
@@ -26,51 +23,4 @@ static UniProtKBEntry convertToUniProtEntry(UniProtEntryObjectProxy objectProxy)
     static InputStream getResourceAsStream(String resourcePath) {
         return TestUtils.class.getResourceAsStream(resourcePath);
     }
-
```
Comment (ahmadshadab) [inline uniprotkb-rest/src/test/java/org/uniprot/api/uniprotkb/controller/TestUtils.java:29]: Boys scout rule. 👍

**Review by ahmadshadab — COMMENTED** [review]
I have one comment about usage of old indexer class UniProtEntryConverter. After your changes, we still have two usage of it first in DataStoreManagerTest and other in UniProtSearchEngine. Should we not use the spark one?

**Comment by LeonardoGonzales** [comment]
UniProtSearchEngine is in Store integration test, it is good to do, but I think it is not the priority now. I created a Jira for it: https://www.ebi.ac.uk/panda/jira/browse/TRM-29305

**Pull request merged**